app/Console/Commands/InstanceUpdateTotalLocalPosts.php

```diff
@@ -68,12 +67,20 @@ protected function getCached()
 
     protected function updateAndCache()
     {
-        $count = DB::table('statuses')->whereNull(['url', 'deleted_at'])->count();
         $res = [
-            'count' => $count,
+            'count' => $this->getTotalLocalPosts(),
         ];
         Storage::put('total_local_posts.json', json_encode($res, JSON_UNESCAPED_SLASHES | JSON_PRETTY_PRINT));
         ConfigCacheService::put('instance.stats.total_local_posts', $res['count']);
 
     }
+
+    protected function getTotalLocalPosts()
+    {
+        return DB::table('statuses')
+            ->whereNull('deleted_at')
+            ->where('local', true)
+            ->whereNot('type', 'share') # Ignore boosts for the post count
```

Comment:
This also could be changed to something else. One thing I was considering was: `->whereNotIn('type', ['share', 'reply', 'story:reply', 'story:reaction'])` to also exclude replies. I don't know that it's worth being especially complicated here. Whatever seems most reasonable we can go with.

Reply:
I believe it makes sense, yes.